webview/jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   collectCoverageFrom: [
     'src/**/*.{ts,tsx}',
     '!**/*.test.*',
-    '!src/stories/*',
+    '!src/stories/**/*.{ts,tsx}',
```

Comment:
Correctly removes the `IconWrapper` and `IconWrappers` components from the coverage.